### ISSUE TITLE
change item need help

### ISSUE THREAD

**Issue by cobrasky3** [InoUno/xi-tinkerer#12]
I have change useableitem ID 4096，name "fire crystal" to "huo shui jing",why client crash when i oppen menu "item"?

**Comment by AlexandreCA** [comment]
yeah good question. I translate in a hurry and I encounter the same problem when I translate a particular word. If anyone knows the answer, I'm interested too.

**Comment by IRedactedI** [comment]
Not sure if this is the same issue, but I get a crash even when nothing is changed. I exported Weapons, created a DAT immediately from the exported yaml without opening it or changing anything, and replaced ROM/118/108.dat. This causes the client to crash when opening the items menu. It also prevents Windower Resource extractor from running, which I assume is for the same reason.

**Comment by InoUno** [comment]
Re-encoding of item DAT files should be fixed now in the latest version (0.3.1).
